### ISSUE TITLE
Do you use reptile rather than maml in training meta-learner?

### ISSUE THREAD

**Issue by machanic** [dydjw9/MetaAttack_ICLR2020#6]
I notice that your code of training meta-learner auto-encoder has an argument `--meta_optim_choose`, like the following code use this argument.
https://github.com/dydjw9/MetaAttack_ICLR2020/blob/master/meta_training/cifar_meta_training/meta.py#L128

But according to your paper, you use reptile.
In order to compare with your approach of compared method, **Should I set `--meta_optim_choose reptile`**?

In other words , should I delete the maml code in my experiments?

**Comment by xiaofanustc** [comment]
yes, you can delete.
